### PR TITLE
[PEN-1461] Quasi revert url compression

### PR DESCRIPTION
**Pull request by jgrosspietsch**
## Description
Added a param for URL compression. I don't remember why at this point.

## Jira Ticket
- [PEN-1461](https://arcpublishing.atlassian.net/browse/PEN-)

## Acceptance Criteria
Yeah, sure.

## Test Steps
Run locally and test images?

### Before
We did URL compression every time.

### After
There's a new param in case you want to opt in to URL compression.

## Dependencies or Side Effects
There's a new param that's only optional.

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps above are working
- [x] Confirmed there are no linter errors
- [x] Confirmed this PR has reasonable code coverage
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

### CODE REVIEW

**Comment by JackHowa** [comment]
Do we have any custom client code that we're trying to fix to test this? Like, to test their implementation that is breaking. Just to sanity check

**Comment by jgrosspietsch** [comment]
> Do we have any custom client code that we're trying to fix to test this?

I have no idea.

**Comment by JackHowa** [comment]
@jgrosspietsch did you find any code we could test this on?

**Comment by JackHowa** [comment]
moved the base branch against beta

File: src/components/Image/SourceHandler.test.tsx [hunk 1]
```diff
@@ -27,11 +30,29 @@ describe('image source handler component', () => {
...
+      compressedParams={false}
     />);
 
     expect(wrapper.html()).not.toBe('<source srcset="www.hey.resizer.com/correct-image.jpg=/100x100/www.hey.com/ffdfdf" media="screen and (min-width: 200px)">');
     expect(wrapper.html()).toBe(null);
     expect(wrapper.text()).toBe('');
     // wanted to use empty render here but not supported in this version of jest, nor on shallow
   });
+
+  describe('when compressedParams is true', () => {
```
Comment (brentmiller1973) [inline src/components/Image/SourceHandler.test.tsx:42]: Awesome

File: src/components/Image/thumbor-image-url.ts [hunk 2]
```diff
@@ -18,7 +19,7 @@ const buildThumborURL = (
    * 3) Convert the default format and quality params to thumbor params.
    */
   let uncompressedTarget = targetImageKeyWithFilter;
-  if (uncompressedTarget.indexOf(':cm=t') !== -1) {
+  if (uncompressedTarget.indexOf(':cm=t') !== -1 && compressedParams) {
```
Comment (brentmiller1973) [inline src/components/Image/thumbor-image-url.ts:22]: Perfect, that's what I was looking for

**Review by brentmiller1973 — APPROVED** [review]

**Pull request merged**